nlp/2021_06_29_long_range_transformers/LongRangeTransformers.ipynb

```diff
@@ -0,0 +1,1940 @@
+       "    <tr style=\"text-align: left;\">\n",
+       "      <th>Step</th>\n",
+       "      <th>Training Loss</th>\n",
+       "      <th>Validation Loss</th>\n",
+       "      <th>Accuracy</th>\n",
+       "      <th>F1</th>\n",
+       "      <th>Precision</th>\n",
+       "      <th>Recall</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>100</td>\n",
+       "      <td>0.694100</td>\n",
+       "      <td>0.691903</td>\n",
+       "      <td>0.489039</td>\n",
+       "      <td>0.656852</td>\n",
+       "      <td>0.489039</td>\n",
+       "      <td>1.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>200</td>\n",
+       "      <td>0.663400</td>\n",
+       "      <td>0.432657</td>\n",
+       "      <td>0.850759</td>\n",
+       "      <td>0.840684</td>\n",
+       "      <td>0.879473</td>\n",
+       "      <td>0.805172</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>300</td>\n",
+       "      <td>0.360200</td>\n",
+       "      <td>0.351535</td>\n",
+       "      <td>0.891231</td>\n",
+       "      <td>0.888118</td>\n",
+       "      <td>0.893543</td>\n",
+       "      <td>0.882759</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>400</td>\n",
+       "      <td>0.286200</td>\n",
+       "      <td>0.443211</td>\n",
+       "      <td>0.887858</td>\n",
+       "      <td>0.879855</td>\n",
+       "      <td>0.924099</td>\n",
+       "      <td>0.839655</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>500</td>\n",
+       "      <td>0.258600</td>\n",
+       "      <td>0.450579</td>\n",
+       "      <td>0.895447</td>\n",
+       "      <td>0.888087</td>\n",
+       "      <td>0.931818</td>\n",
+       "      <td>0.848276</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>600</td>\n",
+       "      <td>0.255000</td>\n",
+       "      <td>0.485321</td>\n",
+       "      <td>0.899663</td>\n",
+       "      <td>0.900251</td>\n",
+       "      <td>0.876020</td>\n",
+       "      <td>0.925862</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table><p>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "tokenizer = RobertaTokenizer.from_pretrained('roberta-base')\n",
+    "\n",
+    "model = RobertaForSequenceClassification.from_pretrained(\n",
+    "    'roberta-base',  gradient_checkpointing=True, num_labels=2)\n",
+    "\n",
+    "run_training(model, tokenizer, 'roberta', train_ds, test_ds, max_length=512)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "k_7w9sa3V-gM"
+   },
+   "source": [
+    "### LongFormer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Njsdd-OaWBpW"
+   },
+   "outputs": [],
```

Comment:
has no output while the other two do, would be nice to harmonize this

Reply:
+1